web/js/components/util/spinner-modal.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Spinner from 'react-loader';
+import { Modal, ModalBody, ModalHeader } from 'reactstrap';
+
+/*
+ * A react component, Builds a rather specific
+ * interactive widget
+ *
+ * @class AnimationWidget
```

Comment:
There are at least a couple `@class AnimationWidget` class names/descriptions throughout for other components

Reply:
grepped through and replaced those ✅